cpp/benchmarks/common/random_distribution_factory.hpp

```diff
@@ -83,7 +83,7 @@ distribution_fn<T> make_distribution(distribution_id did, T lower_bound, T upper
   switch (did) {
     case distribution_id::NORMAL:
       return [lower_bound, dist = make_normal_dist(lower_bound, upper_bound)](
-               std::mt19937& engine) mutable -> T { return dist(engine) - lower_bound; };
+               std::mt19937& engine) mutable -> T { return dist(engine) + lower_bound; };
```

Comment:
I'm curious about this change.  `make_normal_dist` returns a `std::binomial_distribution` from `0` to `(range_end - range_start)`.   The `-lower_bound` seems like it would have been the correct thing to shift the range back into lower_bound/upper_bound. `+lower_bound` seems like it would end up shifting in the wrong direction.

Reply:
First off, I now see that I implemented this in a bad way and need to clean it up and hopefully make clearer. Also, there is an inconsistency between binomial and normal distribution generated here. 

Let's use an example: making a normal integer distribution in range [10, 50].
Created binomial distribution will generate values between 0 and 40 (50-10=40 coin tosses).
We need to add 10 to the generated values to have them be in the [10, 50] range. Since lower bound is 10, having +lower bound seems right to me.
Does this look right? FWIW, it did fix a bug with scale generation for decimal.

Reply:
Hmm.  My internal example was:
`[-10, 10]`

Reply:
Yeah, that was pretty much my internal example as well, so `-lower_bound` seemed correct.
I implemented a tripping hazard :)